test/powershell/engine/Basic/DefaultConfirmImpact.Tests.ps1

```diff
@@ -0,0 +1,256 @@
+            @{ Cmdlet = 'Stop-Computer';                    ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Stop-Job';                         ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Stop-Process';                     ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Stop-Service';                     ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Suspend-Service';                  ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Tee-Object';                       ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Test-Connection';                  ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Test-Json';                        ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Test-ModuleManifest';              ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Test-Path';                        ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Test-PSSessionConfigurationFile';  ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Trace-Command';                    ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Unblock-File';                     ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Unregister-Event';                 ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Unregister-PSSessionConfiguration';ConfirmImpact = 'Low' }
+            @{ Cmdlet = 'Update-FormatData';                ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Update-Help';                      ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Update-TypeData';                  ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Wait-Debugger';                    ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Wait-Event';                       ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Wait-Job';                         ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Wait-Process';                     ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Where-Object';                     ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Write-Debug';                      ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Write-Error';                      ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Write-Host';                       ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Write-Information';                ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Write-Output';                     ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Write-Progress';                   ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Write-Verbose';                    ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Write-Warning';                    ConfirmImpact = 'Medium' }
+        ) # $DefaultCommands
+    }
+
+    It '<Cmdlet> should have a ConfirmImpact rating of <ConfirmImpact>' -TestCases $DefaultCommands {
+        param($Cmdlet, $ConfirmImpact)
+
+        $Type = (Get-Command -Name $Cmdlet).ImplementingType
```

Comment:
Thinking about this, it would be better to just add a new column to the table in https://github.com/PowerShell/PowerShell/blob/master/test/powershell/engine/Basic/DefaultCommands.Tests.ps1 and validate the confirmimpact in that test file

Reply:
If I'm going to have to add to that file, I'd like to overhaul it. The CSV type format is ridiculously hard to read, probably review, add to, and I'm sure it's not fun to maintain.

Most of these sorts of tests that I've seen thus far opt for an array of hashtables rather than a big block of CSV text. I feel like that might also be a better option here?

Reply:
We could merge these tests. 
I think we don't improve readability if we convert to hashtable - we get very long line or we'll be forced to collapse to multiline.
So I'd simply add the new column.

Reply:
Sounds good... The best way for me to do this would be programmatically from a current version of PS, but that'll butcher the spacing... I suppose I can just add that back in... 🤷‍♂️ yeah it'll work.

Reply:
VS Code support to edit _columns_.